scripts/upgrade

```diff
@@ -11,45 +11,67 @@ source /usr/share/yunohost/helpers
 #=================================================
 # LOAD SETTINGS
 #=================================================
-ynh_script_progression --message="Loading installation settings..." --weight=2
+ynh_script_progression --message="Loading installation settings..." --time --weight=1
```

Comment:
You can remove all those --time arguments

Reply:
Sure, I forgot to remove them 😅